templates/webapps/galaxy/workflow/sharing.mako

```diff
@@ -145,7 +145,9 @@
                             ${"/".join( url_parts[:-1] )}/<span id='item-identifier'>${url_parts[-1]}</span>
                         </span>
 
-                        <a href="javascript:void(0)" role="button" id="edit-identifier"><img src="${h.url_for('/static/images/fugue/pencil.png')}"/></a>
+                        <a href="javascript:void(0)" role="button" id="edit-identifier">
```

Comment:
since we're here already, should we ```href="javascript:void(0)"``` => `cursor: pointer;` ?

Reply:
Given this is an old mako and targeting a stable release, let's just handle that more gracefully in a modern component?  If you want to push another commit to this branch though, feel free.

Reply:
> targeting a stable release, let's just handle that more gracefully 

:+1: